sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -506,20 +533,23 @@ async Task GetScopeProperties (MessageId msg_id, int scope_id, CancellationToken
 
 				var values = res.Value? ["result"]? ["value"]?.Values<JObject> ().ToArray ();
 
-				var var_list = new List<JObject> ();
+				if(values == null)
+					SendResponse (msg_id, Result.OkFromObject (new {result = Array.Empty<object> ()}), token);
```

Comment:
Do we need to return from here? Rest of the method assumes that `values != null`.

Reply:
I'd rather exit here than check in both of the loops below.  This was an actual case I hit in the second loop.